holmes/plugins/toolsets/bash/common/cli_prefixes.py

```
@@ -66,24 +198,15 @@ def save_cli_bash_tools_approved_prefixes(prefixes: List[str]) -> None:
-        try:
-            with open(prefixes_file, "r") as f:
-                data = yaml.safe_load(f)
-                if isinstance(data, dict) and "approved_prefixes" in data:
-                    existing = set(data["approved_prefixes"])
-        except Exception:
-            pass
-
-    updated = sorted(set(prefixes) | existing)
+    try:
+        existing = load_approved_prefixes_file(prefixes_file)
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)